phone/src/apps/messages/components/modal/MessageImageModal.tsx

```diff
@@ -27,6 +29,11 @@ export const MessageImageModal = ({ isOpen, messageGroupId, onClose, image }: IP
   const [pasteVisible, setPasteVisible] = useState(false);
   const [queryParamImagePreview, setQueryParamImagePreview] = useState(null);
 
+  const isImage = (url) => {
```

Comment:
This may be better in `utils/isImage.ts` as its used in more than 2 places I think